evm.md

```diff
@@ -1501,8 +1500,7 @@ For each `CALL*` operation, we make a corresponding call to `#call` and a state-
          <id> ACCT => ACCTTO </id>
          <gas> _ => GCALL </gas>
          <callGas> GCALL => 0 </callGas>
-         <program> _ => #asMapOpCodes(#dasmOpCodes(INITCODE, SCHED)) </program>
-         <programBytes> _ => INITCODE </programBytes>
+         <program> _ => INITCODE </program>
```

Comment:
Let's `#computeValidJumpDests` here as well.

Reply:
Unless you want to move program loading into `initVM` by passing it as an argument, which would reduce rule duplication but also make sure that the `<jumpDests>` and `<program>` cells always agree.